dot/state/pruner/pruner.go

```diff
@@ -110,7 +109,7 @@ func newJournalRecord(hash common.Hash, insertedMerkleValues,
 }
 
 // NewFullNode creates a Pruner for full node.
-func NewFullNode(db, storageDB chaindb.Database, retainBlocks uint32, l log.LeveledLogger) (Pruner, error) {
+func NewFullNode(db, storageDB chaindb.Database, retainBlocks uint32, l Logger) (Pruner, error) {
```

Comment:
The pruner and chaindb.Database usage is quite changed in #2831 so it's left unchanged here to reduce conflicts.